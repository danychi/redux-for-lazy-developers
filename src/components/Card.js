import styled from 'styled-components';

export default styled.div`
  border-radius: 3px;
  border: 1px solid #efefef;
  background-color: #fff;
  box-shadow: 0 1px 4px rgba(0, 0, 0, 0.04);
`;
