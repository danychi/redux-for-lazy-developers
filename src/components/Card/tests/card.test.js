import React from 'react';
import renderer from 'react-test-renderer';
import Card from '../index';

describe('Card', () => {
  it('renders successfully with the required props', () => {
    const tree = renderer.create(<Card>Test</Card>);
    expect(tree).toMatchSnapshot();
  });
});
