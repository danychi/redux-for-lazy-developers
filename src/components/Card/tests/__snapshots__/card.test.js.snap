// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card renders successfully with the required props 1`] = `
.c0 {
  min-width: 152px;
}

<div
  className="c0"
>
  Test
</div>
`;
